tests/Feature/Commands/InstallBaseStructureCommandTest.php

```diff
@@ -2,6 +2,11 @@
 
 use Illuminate\Support\Facades\Artisan;
 
+beforeEach(closure: function () {
+    (new Illuminate\Filesystem\Filesystem())
```

Comment:
Put in Pest and use it in every test would be great

Reply:
it means for each test in pest, before i test it i make sure to clear the dir seems another test break it

Reply:
We can specific witch file will use this deletetion of directory